packages/react-storage/src/styles/storage-browser.css

```diff
@@ -529,3 +530,8 @@
 .storage-browser__search__submit-button:not(:focus) {
   border-inline-start-color: transparent;
 }
+
+.storage-browser__locations-view-search,
```

Comment:
Maybe just leave CSS out for now, Danny overhauled a lot of the CSS in my copy view pr.